vcd/resource_vcd_vapp.go

```diff
@@ -452,10 +452,10 @@ func getVAppIPAddress(d *schema.ResourceData, meta interface{}, vdc govcd.Vdc, o
 	vcdClient := meta.(*VCDClient)
 	var ip string
 	err := retryCall(vcdClient.MaxRetryTimeout, func() *resource.RetryError {
-		err := vdc.Refresh()
-		if err != nil {
-			return resource.RetryableError(fmt.Errorf("error refreshing VDC: %#v", err))
-		}
+		// err := vdc.Refresh()
```

Comment:
Why we don't need vdc.Refresh() here?

Reply:
`vdc.FindVAppByName` already has an implicit Refresh